plugin/src/main/resources/META-INF/fabric8/profiles-default.yml

```diff
@@ -24,6 +24,7 @@
     includes:
     - fmp-name
     - fmp-controller
+    - fmp-pluginConfiguration-controller
```

Comment:
can we avaoid to introduce camel case for enricher names ? I just would call it `fmp-controller-from-configuration` or so.